src/preamble.js

```diff
@@ -1129,16 +1130,16 @@ function createWasm() {
 #endif
 #if MODULARIZE
   // If instantiation fails, reject the module ready promise.
-  instantiateAsync(wasmBinary, wasmBinaryFile, info, receiveInstantiationResult).catch(readyPromiseReject);
+  instantiateAsync(wasmBinary, wasmBinaryFile, imports, receiveInstantiationResult).catch(readyPromiseReject);
```

Comment:
If we are making the `imports` object global, then perhaps we just remove the third argument from this function completely?

Reply:
found it used elsewhere

https://github.com/emscripten-core/emscripten/blob/0a6f54b1276481fab90e474f2c4fd35d00c5fa6b/src/library_async.js#L541-L550